networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/rpc/core/Eth2OutgoingRequestHandlerTest.java

```diff
@@ -280,7 +282,7 @@ public void shouldCompleteExceptionallyWhenClosedWithTruncatedMessage() {
 
     asyncRequestRunner.executeQueuedActions();
 
-    complete();
+    assertThrows(IllegalReferenceCountException.class, this::complete);
```

Comment:
We shouldn't be expecting this and it probably shouldn't be in this change.  Need to work out why it's being released twice and fix that in a different PR. :)

Reply:
alright but it blocks every build i do, so basically i cant get a green build on my machine before i push

Reply:
I don't think this is the right fix either - complete exceptionally is different to throws exception.  I wouldn't have expected to get this exception from a call to `complete`.